pkg/aws/services/vpclattice.go

```diff
@@ -26,6 +26,20 @@ type LatticeServiceNameProvider interface {
 	LatticeServiceName() string
 }
 
+type defaultLatticeServiceNameProvider struct {
+	name string
+}
+
+func NewLatticeServiceNameProvider(name string) LatticeServiceNameProvider {
```

Comment:
Should be `NewDefaultLatticeServiceNameProvider `?  LatticeServiceNameProvider is a interface name